### PR TITLE
added annotations import to network binding

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

**Comment by martinvw** [comment]
@kaikreuzer travis is not happy....

**Comment by kaikreuzer** [comment]
I know, see https://github.com/openhab/openhab2-addons/issues/2867#issuecomment-346614135

**Comment by kaikreuzer** [comment]
Now it is happy and we need this fix to make the other builds happy, too!

**Comment by kaikreuzer** [comment]
Please allow me to self-merge...

File: addons/binding/org.openhab.binding.network/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -27,5 +27,6 @@ Import-Package:
  org.openhab.binding.network,
  org.openhab.binding.network.handler,
  org.osgi.service.component,
+ org.osgi.service.component.annotations;resolution:=optional,
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.network/META-INF/MANIFEST.MF]: Should this also be reverted https://github.com/openhab/openhab-core/pull/237

Comment (kaikreuzer) [reply addons/binding/org.openhab.binding.network/META-INF/MANIFEST.MF]: yes.

**Pull request merged**